media/js/app/taxonomy/taxonomy.js

```diff
@@ -474,16 +494,47 @@
                 return item.display_name == thing;
             });
         },
-        getTheOnomy: function(onomyURL, selectedVocabulary) {
+        getTheOnomy: function(onomyURL, selectedVocabulary, JSON_FLAG) {
```

Comment:
regular javascript variable names should be described in camelCase. (though we are guilty of mixing in the python underbar syntax!)  This all uppercase variable name might be confused with a global constant.